src/YPerl.cc

```diff
@@ -142,6 +145,24 @@ YPerl::yPerl()
     return _yPerl;
 }
 
+
+void YPerl::fixupLocale()
+{
+    y2milestone( "Switching to the global locale" );
+
+    uselocale( LC_GLOBAL_LOCALE ); // bsc#1216689
+
+    // Those functions only query the current values,
+    // they don't change anything.
+
+    char * locale  = setlocale( LC_ALL, "" );
```

Comment:
That should be NULL instead of "".

Reply:
Ah, OK. `""` is also a valid argument here, but then all will be set according to the environment variables, I just read. Good point.